pydantic/main.py

```diff
@@ -169,6 +171,60 @@ def validate_custom_root_type(fields: Dict[str, Field]) -> None:
 TYPE_BLACKLIST = FunctionType, property, type, classmethod, staticmethod
 
 
+class ValueExclude:
+    __slots__ = ('__exclude__', '__type__')
```

Comment:
```suggestion
    __slots__ = ('_exclude', '_type')
```

Reply:
Why? :thinking:

Reply:
because `_foobar` is standard for naming private properties, `__foobar__` is generally kept for special properties.

Reply:
You're right. Thanks!